### ISSUE TITLE
Add existing curated boosters

### ISSUE THREAD

**Issue by LFP6** [EteRNAgame/curated-boosters#4]
Naive Solver, Sequence Stamper, and Script Executor should probably be added here.

**Comment by Omei** [comment]
Agreed.  Let's get the first new one under our belt, and then I'll propose to the dev team that players take over responsibility for the others.
